sdk/metricsadvisor/Azure.AI.MetricsAdvisor/tests/MetricsAdvisorClient/TimeSeriesLiveTests.cs

```diff
@@ -27,7 +27,7 @@ public async Task GetDimensionValuesWithMinimumSetup(bool useTokenCredential)
 
```

Comment:
are you thinking of renaming the test names too?

Reply:
Forgot the tests. Will update them.